code/modules/research/designs/autolathe_designs.dm

```diff
@@ -5,18 +5,20 @@
 /datum/design/bucket
 	name = "Bucket"
 	id = "bucket"
-	build_type = AUTOLATHE
+	build_type = AUTOLATHE | PROTOLATHE
 	materials = list(MAT_METAL = 200)
 	build_path = /obj/item/reagent_containers/glass/bucket
-	category = list("initial","Tools")
+	category = list("initial","Tools","Equipment")
```

Comment:
Should this be `category = list("initial","Tools","Tool Designs")` ?

Reply:
Yeah, that would make sense.